packages/meso-js/test/transfer.test.ts

```diff
@@ -66,14 +65,15 @@ describe("transfer", () => {
       [
         "https://api.sandbox.meso.network",
         {
+          "authenticationStrategy": "wallet_verification",
           "destinationAsset": "ETH",
           "headlessSignature": "false",
           "layoutOffset": "0",
           "layoutPosition": "top-right",
           "network": "eip155:1",
           "partnerId": "partnerId",
           "sourceAmount": "100",
-          "version": "${version}",
```

Comment:
We should keep this string interpolation so that this test doesn't fail on every version bump.